INSTALLING.md

```diff
@@ -3,61 +3,76 @@ Installing Ion Auth.
 
 Before installing, please check that you are meeting the minimum server requirements.
 
-There are 2 ways to install this package.
+There are 3 ways to install this package.
 
-> 1. Move files from this package to the corresponding folder structure:
+> 1. Move folder from this package to the root folder:
 
 ```shell
 CI                          # → Root Directory
 └── application/
-    ├── config/
-    │   └── ion_auth.php
-    ├── controllers/
-    │   └── Auth.php
-    ├── libraries
-    │   └── Ion_auth.php
-    ├── models
-    │   └── Ion_auth_model.php
-    └── views/
-        └── auth/           # → Various view files
+├── ion-auth/               # → Ion-auth directory
+├── public
+├──...
+```
+Then in your Config/Autoload.php, add this :
+```php
+'IonAuth' => ROOTPATH . 'YOUR-ION_AUTH-FOLDER',
 ```
-
 
 ---
 
-
-> 2. Move files from this package to the corresponding **third_party** structure:
+> 2. With Git:
 
 ```shell
+my-project$ git clone https://github.com/benedmunds/CodeIgniter-Ion-Auth.git
+my-project$ git checkout 4
+```
+Then in your Config/Autoload.php, add this :
+```php
+'IonAuth' => ROOTPATH . 'Code-Igniter-Ion-Auth',
+```
 CI                          # → Root Directory
 └── application/
-    ├── controllers/
-    │   └── Auth.php
-    ├── third_party/
-    │   └── ion_auth/
-    │       ├── config/
-    │       │   └── ion_auth.php
-    │       ├── libraries
-    │       │    └── Ion_auth.php
-    │       └── models
-    │            └── Ion_auth_model.php
-    └── views/
-        └── auth/           # → Various view files
+├── ion-auth/               # → Ion-auth directory
+├── public
+├──...
+```
+Then in your Config/Autoload.php, add this :
+```php
+'IonAuth' => ROOTPATH . 'YOUR-ION_AUTH-FOLDER',
 ```
 
-Then in your controller, example `Auth.php` add the package path and load the library like normal
 
-	$this->load->add_package_path(APPPATH.'third_party/ion_auth/');
-	$this->load->library('ion_auth’);
+---
 
-Or autoload by adding the following to application/config/autoload.php
 
-	$autoload['packages'] = array(APPPATH.'third_party/ion_auth');
+> 3. Install it with composer
```

Comment:
Thoughts on making composer the 1st choice, git the 2nd, and copying files last?  That way its really in the method most modern projects use.

Reply:
You're right. Done !